src/ast/codegen_llvm.cpp

```diff
@@ -1294,6 +1294,87 @@ void CodegenLLVM::binop_int(Binop &binop)
   expr_ = b_.CreateIntCast(expr_, b_.getInt64Ty(), false);
 }
 
+void CodegenLLVM::binop_ptr(Binop &binop)
+{
+  auto compare = false;
+  auto arith = false;
+
+  // Do what C does
+  switch (binop.op)
+  {
+    case bpftrace::Parser::token::EQ:
+    case bpftrace::Parser::token::NE:
+    case bpftrace::Parser::token::LE:
+    case bpftrace::Parser::token::GE:
+    case bpftrace::Parser::token::LT:
+    case bpftrace::Parser::token::GT:
+      compare = true;
+      break;
+    case bpftrace::Parser::token::LEFT:
+    case bpftrace::Parser::token::RIGHT:
+    case bpftrace::Parser::token::MOD:
+    case bpftrace::Parser::token::BAND:
+    case bpftrace::Parser::token::BOR:
+    case bpftrace::Parser::token::BXOR:
+    case bpftrace::Parser::token::MUL:
+    case bpftrace::Parser::token::DIV:
+      break;
```

Comment:
maybe `assert(false)` here? Might help catch bugs early if anything changes in future